setup.py

```diff
@@ -0,0 +1,53 @@
+from setuptools import setup, find_packages
+
+def main():
+    setup(
+        name='deepspeech_training',
+        version='0.0.1',
```

Comment:
Shouldn't we set proper version here?

Reply:
Our policy has been that training code is not part of the stable API covered by semver, so I thought synchronizing the versions would give the wrong impression. There's no intention of publishing this package, it's just for clean dependency management.

Reply:
Sure, but it's not going to be long before people ask for it. Also, there might still be value in doing that to make it easier to debug certain things: `pip list` would be more consistent for example.

Reply:
OK, I'll sync the versions then. As for PyPI, I don't think we should publish the training code there even if people ask for it.

Reply:
I'm unsure about PyPI as well, it could be a nest of problems. I remember some people wanted to do that. But in the current state, it still raises a lot of questions: like how do we package `data/` (it's big, it might not make sense, it will likely be a source of troubles for everyone to upload, host and download).

Reply:
Yeah, exactly, how to handle `data` is a big concern. We want to take that out of the repo anyway as we add more languages, so in the future we can revisit this.